build.gradle

```diff
@@ -70,3 +71,17 @@ allprojects {
 		}
 	}
 }
+
+apply plugin: 'binary-compatibility-validator'
```

Comment:
Lets do one of two things:

- Apply it directly to the projects that we want (runtime, Gradle plugin)
- Apply it to all projects which have the publishing plugin applied. The implication being if we are publishing the artifact then we care about the API.